python/kserve/kserve/protocol/rest/openai/config.py

```diff
@@ -42,9 +32,6 @@ def get_open_ai_models(repository: ModelRepository) -> List[Model]:
 
 
 def maybe_register_openai_endpoints(app: FastAPI, model_registry: ModelRepository):
-    # Check if the openai package is available before continuing so we don't run into any import errors
-    if not openai_is_available():
-        return
     open_ai_models = get_open_ai_models(model_registry)
     # If no OpenAI models then no need to add the endpoints
     if len(open_ai_models) == 0:
```

Comment:
@yuzisun we only register the OpenAI endpoints if one of more of the registered models extends `OpenAIModel`